src/api/ecom-api.tsx

```diff
@@ -51,11 +51,22 @@ function createApi(): EcomAPI {
                 if (!category) {
                     throw new Error('Category not found');
                 }
-                const productsResponse = await wixClient.products
+                let productsResponse = await wixClient.products
                     .queryProducts()
                     .hasSome('collectionIds', [category!._id])
+                    .limit(100)
```

Comment:
do we need to set this limit? I guess default limit is 100

Reply:
no, default is 50, but 100 is a maximum allowed. API crashes if set more then 100